docs/UserGuide.adoc

```diff
@@ -327,16 +327,16 @@ To quickly revert address book data to the state of last use:
 
 * *Login* `login USERNAME PASSWORD` +
 e.g. `login userAcc_123 pa$$_Word!@#&`
-* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS pc/POSTAL CODE d/DEBT [t/TAG]...` +
-e.g. `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd pc/123466 d/123 t/friend t/colleague`
+* *Add* `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS pc/POSTAL CODE d/DEBT dl/DEAD LINE[t/TAG]...` +
```

Comment:
DEADLINE* ? Unsure if this is intentional

Reply:
fixed